src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/AuthorizationRule/SetAzureEventhubAuthorizationRules.cs

```diff
@@ -0,0 +1,115 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.EventHub.Models;
+using Microsoft.Azure.Management.EventHub.Models;
+using System.Management.Automation;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Commands.EventHub.Commands
+{
+    /// <summary>
+    /// 'Set-AzureRmRelayAuthorizationRule' Cmdlet updates the specified AuthorizationRule
+    /// </summary>
+    [Cmdlet(VerbsCommon.Set, EventHubAuthorizationRuleVerb, DefaultParameterSetName = NamespaceAuthoRuleParameterSet, SupportsShouldProcess = true), OutputType(typeof(SharedAccessAuthorizationRuleAttributes))]
+    public class SetAzureRelayAuthorizationRule : AzureEventHubsCmdletBase
+    {
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 0,
+            HelpMessage = "Resource Group Name.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 1, ParameterSetName = NamespaceAuthoRuleParameterSet,
+            HelpMessage = "Namespace Name.")]
+        [Parameter(ParameterSetName = EventhubAuthoRuleParameterSet)]
```

Comment:
@v-Ajnava please provide parameter attributes for the extra parameter sets you add (_e.g.,_ there is nothing about if this parameter is mandatory or not, what it's position is, if it accepts it's value from pipeline, etc. in this parameter set)

This comment applies to all parameters missing these attributes across cmdlets.